docs/content/preview/yugabyte-platform/security/enable-encryption-at-rest.md

```diff
@@ -110,6 +110,6 @@ To rotate the universe keys, perform the following:
 You can disable encryption at rest for a universe as follows:
 
 1. Open the YugabyteDB Anywhere UI and navigate to the universe for which to disable encryption.
-2. Select **Action > Edit Universe > Edit Security > Encryption-at-Rest**.
+2. Select **Action > Edit Security > Encryption-at-Rest**.
```

Comment:
Action or Actions (see Rotate and enable above)?

Reply:
Fixed